update_rules_content.sh

```diff
@@ -31,7 +36,7 @@ RULES_CONTENT="${CLONE_TEMP_DIR}/content/"
 
 echo "Attempting to clone repository into ${CLONE_TEMP_DIR}"
 
-if ! git clone --depth=1 --branch master "${RULES_REPO}" "${CLONE_TEMP_DIR}"
+if ! git clone --depth=1 --branch "${CCX_RULES_OCP_TAG}" "${RULES_REPO}" "${CLONE_TEMP_DIR}"
 then
     echo "Couldn't clone rules repository"
     exit 1
```

Comment:
minor thing but if we replace `exit 1` with `exit $?` we can get the exit status of the command that failed